securedrop_client/export.py

```diff
@@ -21,10 +21,23 @@ def __init__(self, status: str):
 
 
 class ExportStatus(Enum):
+    """
+    Status codes representing results of export- and print- related calls.
+
+    Codes may represent failure states (`USB_BAD_PASSPHRASE`) or success states
+    (`USB_CONNECTED`).
+
+    Warning: do not make changes to existing values without reviewing
+    `securedrop-export/securedrop_export/export.py`.
+    """
+
     # On the way to success
     USB_CONNECTED = "USB_CONNECTED"
     DISK_ENCRYPTED = "USB_ENCRYPTED"
 
+    # (Success) Drive is compatible and unlocked; do not prompt for passphrase
+    USB_ENCRYPTED_UNLOCKED = "USB_ENCRYPTED_UNLOCKED"
+
```

Comment:
as mentioned in the PR, in the future we'll refine this to return multiple status codes (`LUKS_UNLOCKED`, `VERACRYPT_UNLOCKED`)